CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## next
+
+### NEW - option `--output-name`
+
+It is possible to specify which constant or function you would like to output by adding `--output-name=myCustomConstantOrFunction`.
```

Comment:
Specify a custom name for the output constant or function by adding `--output-name=myCustomConstantOrFunction`.